.agents/skills/cuopt-server-api-python/assets/lp_basic/client.py

```
@@ -0,0 +1,84 @@
+        },
+        "variable_bounds": {
+            "upper_bounds": ["inf", "inf"],
+            "lower_bounds": [0.0, 0.0],
+        },
+        "maximize": True,
+        "solver_config": {
+            "time_limit": 60,
+        },
+    }
+
+    response = requests.post(
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)